xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3703,7 +3707,7 @@ tsl::StatusOr<cudnn_frontend::Tensor> CreateCudnnMaskTensor(
   // Create the mask output tensor
   TF_ASSIGN_OR_RETURN(
       auto mask_out_tensor,
-      CreateCudnnTensor(dims, strides, CudnnfMHAUid::VIRTUAL_ID + 300,
+      CreateCudnnTensor(dims, strides, CudnnfMHAUid::VIRTUAL_ID + 400,
```

Comment:
Could you give some context for magic constants?

Reply:
Every tensor in the graph needs to have a unique ID, VIRTUAL_ID is just some large number to begin with so it does not overlap with the non-virtual tensor defined in `enum CudnnfMHAUid`. Tensor ID in scale/bias/mask/softmax/dropout will start with +200, +300, +400, +500, +600 respectively.